src/components/routes/Host/ListingForm/ListingHelpText.tsx

```diff
@@ -0,0 +1,232 @@
+import * as React from 'react';
+import ListItem from 'shared/ListItem';
+import Svg from 'shared/Svg';
+import LazyImage from 'shared/LazyImage';
+
+interface ListingHelpTextInterfaceInterface {
+  [name: string]: React.ReactNode;
```

Comment:
oops `InterfaceInterface` was from me being vscode trigger happy. good catch!

Reply:
condensed filename to `ListingHelp`

tried to use the enum in building the interface but typescript doesn't wanna do it
![image](https://user-images.githubusercontent.com/18321302/52316571-d3eb4d00-2970-11e9-8363-f7e7faa45dba.png)

Looked at few sources and it might require some golfing.
https://github.com/Microsoft/TypeScript/issues/5683
https://stackoverflow.com/questions/50849359/how-can-i-use-a-string-enum-type-as-a-computed-property-name-in-an-interface